e2e/pages/adf/dataTablePage.ts

```diff
@@ -38,6 +38,7 @@ export class DataTablePage {
     tableBody = element.all(by.css(`div[class='adf-datatable-body']`)).first();
     spinner = element(by.css('mat-progress-spinner'));
     rows = by.css(`adf-datatable div[class*='adf-datatable-body'] div[class*='adf-datatable-row']`);
+    permissionsDataTable = element(by.css("adf-datatable[class*='adf-datatable-permission']"));
```

Comment:
This is not a common datatable locator, 'adf-datatable-permission' is specific to demo-shell or another component.  The datatable has a constructor, so you can give as root 'adf-permission-list' and then use the datatable normally. Some examples can be found in contentServicesPage:     getUploadAreaDocumentList() {
        return new ContentListPage(element(by.css('adf-upload-drag-area')));
    } (well, this uses ContentListPage, but the iea is the same), tasksListPage.ts, taskListCloudComponent, processListCloudComponent